### PR TITLE
[SCRIPTS ONLY] .bat script to use github token instead of user/password

### PR DESCRIPTION
**Pull request by ochaloup**

### CODE REVIEW

File: scripts/hudson/quickstart.bat [hunk 1]
```diff
@@ -115,6 +115,6 @@ goto:eof
    for /f "tokens=1,2,3,4 delims=/" %%a in ("%GIT_BRANCH%") do set IS_PULL=%%b&set PULL_NUM=%%c
    if not "%IS_PULL%"=="pull" goto:eof
    
-   curl -k -d "{ \"body\": \"%~1\" }" -ujbosstm-bot:%BOT_PASSWORD% https://api.github.com/repos/%GIT_ACCOUNT%/%GIT_REPO%/issues/%PULL_NUM%/comments
+   curl -k -d "{ \"body\": \"%~1\" }" -H "Authorization: token %GITHUB_TOKEN%" https://api.github.com/repos/%GIT_ACCOUNT%/%GIT_REPO%/issues/%PULL_NUM%/comments
```
Comment (mayankkunwar) [inline scripts/hudson/quickstart.bat:118]: Here who is setting `GITHUB_TOKEN` ?

Comment (mayankkunwar) [inline scripts/hudson/quickstart.bat:118]: Is the value of `GITHUB_TOKEN` constant or variable?

Comment (ochaloup) [reply scripts/hudson/quickstart.bat:118]: The `GITHUB_TOKEN` is taken from the CI system. Take a look on the guide at https://github.com/jbosstm/narayana/wiki/Jenkins-CI-and-PR-comments-setup

Comment (ochaloup) [reply scripts/hudson/quickstart.bat:118]: The value of `GITHUB_TOKEN` is variable. It's propagated from the Jenkins CI.

Comment (mayankkunwar) [reply scripts/hudson/quickstart.bat:118]: Okay, thanks for the reference.

Comment (mayankkunwar) [reply scripts/hudson/quickstart.bat:118]: That makes sense.

**Pull request merged**